setup.py

```diff
@@ -44,7 +44,7 @@ def find_version(*file_paths):
     'sentencepiece==0.1.91',
     'protobuf',
     'pandas',
-    'tokenizers==0.8.1',
+    'tokenizers==0.9.4',
```

Comment:
Does it also work for 0.9.3?

Reply:
I just checked that huggingface has upgraded to 0.9.4 also. So there is no need to support 0.9.3 anymore.